.travis.yml

```diff
@@ -32,6 +30,13 @@ env:
     - INSTALL_COMPOSER: false
     - INSTALL_WORDPRESS: true
 
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - libstdc++-4.9-dev
```

Comment:
Can you explain what this is for? Is it possible to leave an inline comment with said explanation?

Reply:
NodeGit breaks with the default C lib that Travis provides now.

I added a comment.